frontend/js/src/utils/types.d.ts

```diff
@@ -682,3 +682,7 @@ type UserFreshReleasesResponse = {
 declare type SearchUser = {
   user_name: string;
 };
+
+declare type UserPreferences = {
+  saveDate?: boolean
```

Comment:
This is misnamed in various places in the PR
```suggestion
  saveData?: boolean
```

Reply:
Fixed in all places